src/styles.css/style.css

```diff
@@ -0,0 +1,118 @@
+@import "./stars.css";
+@import "./reset.css";
+
+html {
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.card {
```

Comment:
You should use a padding for this card not only for top side, but other sides as well
In this way you will not to set padding for such selectors as `.card__model`, `.card__item-code` etc